docs/how-to/migrate/forkid-7-to-9.md

```diff
@@ -0,0 +1,286 @@
+    cast send -j --rpc-url "http://$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)" \
+        --private-key 0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625 \
+        $mngr \
+        'updateRollup(address,uint32,bytes)' \
+        "$rollup" 2 0x > update-rollup-type-out.json
+    ```
+
+7. Verify the updated rollupid. Previously the 4th value was a `7` and now it should be a `9`.
+
+    ```sh
+    cast call --rpc-url "$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)" \
+        "$(jq -r '.L1Config.polygonRollupManagerAddress' /tmp/fork-7-test/genesis.json)" \
+        "rollupIDToRollupData(uint32)(address,uint64,address,uint64,bytes32,uint64,uint64,uint64,uint64,uint64,uint64,uint8)" 1
+    ```
+
+8. Set up the data availability protcol again:
+
+    ```sh
+    rollup="0x1Fe038B54aeBf558638CA51C91bC8cCa06609e91"
+    dac="0x5A6896A98c4B7C7E8f16d177C719a1d856b9154c"
+    cast send -j \
+        --private-key "0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625" \
+        --rpc-url "$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)" \
+        "$rollup" 'setDataAvailabilityProtocol(address)' $dac > set-dac-out.json
+    ```
+
+## Node upgrade
+
+At this stage, the smart contracts are upgraded. However, we still need to start the nodes again. 
+
+!!! warning
+    - This procedure is very sensitive.
+    - Ensure the synchronizer starts first. 
+    
+We're going to revert the parameters back to the versions of the node that worked with fork 9, and specify that _ONLY_ stage 3 should run.
+
+1. Update the `params.yml` file as follows:
+
+    ```sh
+    diff --git a/params.yml b/params.yml
+    index c2dd446..cdb8338 100644
+    --- a/params.yml
+    +++ b/params.yml
+    @@ -8,7 +8,7 @@ deployment_suffix: "-001"
+    # The deployment process is divided into various stages.
+    # The `stages` parameter indicates the specific stages you wish the deployment to proceed through.
+    # By default, it will execute all the stages.
+    -stages: [1, 2, 3, 4, 5]
+    +stages: [3]
+
+    # Docker images and repositories used to spin up services.
+    zkevm_prover_image: hermeznetwork/zkevm-prover:v6.0.0
+    ```
+
+2. Remove the `HaltOnBatchNumber` setting that we added earlier.
+
+3. Run Kurtosis to bring up the main node components. 
+      
+    ```sh
+    kurtosis run --enclave cdk-v1 --args-file params.yml --image-download always .
+    ```
+
+4. The core services are now running and we should be able to send a transaction and see the batch numbers moving through their normal progression.
+
+    ```sh
+    export ETH_RPC_URL="$(kurtosis port print cdk-v1 zkevm-node-rpc-001 http-rpc)"
+    cast send --legacy --private-key 0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625 --value 0.01ether 0x0000000000000000000000000000000000000000
+    cast rpc zkevm_batchNumber
+    cast rpc zkevm_virtualBatchNumber
+    cast rpc zkevm_verifiedBatchNumber
+    ```
+
+## Troubleshooting
```

Comment:
Also mention that users cloned the `zkevm-contracts` repository in the same folder as `kurtosis-cdk`, they may see this error when trying to deploy the stack. The solution is to move the `zkevm-contracts` folder outside of the `kurtosis-cdk` folder.

```bash
Error:  An error occurred running command 'run'
  Caused by: An error occurred calling the run function for command 'run'
  Caused by: An error starting the Kurtosis code execution '.'
  Caused by: Error uploading package '.' prior to executing it
  Caused by: There was an error compressing module '.' before upload
  Caused by: An error occurred creating the archive from the files at '.'
  Caused by: The files you are trying to upload, which are now compressed, exceed or reach 100mb. Manipulation (i.e. uploads or downloads) of files larger than 100mb is currently disallowed in Kurtosis.
```